source/sass/blocks/page-main.scss

```
@@ -11,18 +11,35 @@
   background-position: center center;
   background-repeat: no-repeat;
 
-  @media (max-width: 320px) {
+  @media (max-width: $width-mobile) {
     background-image: url("../img/main-background-mobile.jpg");
   }
 
-  @media (min-width: 321px) and (max-width: $width-tablet) {
+  @media (min-width: 321px) and (max-width: 767px) {
     background-image: url("../img/main-background-tablet.jpg");
   }
 
-  @media (min-width: $width-desktop) {
+  @media (min-width: $width-tablet) {
     background-image: url("../img/main-background-desktop.jpg");
     height: 518px;
   }
+
+  &--short {
+    @media (max-width: $width-mobile-only) {
+      background-position: center -231px;
```

```suggestion
      background-position: center bottom;
      background-size: cover;
```
Чтобы не переопределять положение фона на каждой ширине используй **bottom**
А с **background-size: cover** фон будет тянуться и белые углы снизу сохранятся на мобильных